presets/4.3/tune/fpvian_basher_freestyle.txt

```diff
@@ -0,0 +1,181 @@
+#$ TITLE: Basher Freestyle 5" by FPVian
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: basher, freestyle, 5 Inch, 5 In, 5", bando, fpvian
+#$ AUTHOR: FPVian
+#$ DESCRIPTION: Want to be able to take your quad to the BANDO and still have it fly well?
+#$ DESCRIPTION: Want a preset that you can load on to all your 5" Freestyle quads?
+#$ DESCRIPTION: Don't want to give up the feeling of flying a well-tuned machine?
+#$ DESCRIPTION:
+#$ DESCRIPTION: Well look no further! Guaranteed to contain a filter preset that works for you.
+#$ DESCRIPTION: Don't like it? Go fix your bashed-up quad!
+#$ DESCRIPTION: 
+#$ DESCRIPTION: Optimized for AUW: ~700g (including HD cam), but anything should work fine.
+#$ DESCRIPTION: The "Filtering for CLEAN builds" option is recommended for best overall performance.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Leaving all filter options unchecked will leave the filters unchanged.
+#$ DESCRIPTION: You MUST enable bidirectional DSHOT300 or DSHOT600 to use the filtering options.
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+#$ DISCUSSION: 
```

Comment:
`#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/166`

Reply:
Added PR link, thanks for the reminder!